internal/services/dataprotection/data_protection_backup_instance_postgresql_resource.go

```diff
@@ -68,6 +69,12 @@ func resourceDataProtectionBackupInstancePostgreSQL() *pluginsdk.Resource {
 				Required:     true,
 				ValidateFunc: validate.BackupPolicyID,
 			},
+
+			"database_credential_key_vault_secret_id": {
```

Comment:
This could be suffixed to match the attribute from the key_vault_secret resource

```suggestion
			"database_credential_key_vault_secret_versionless_id": {
```